stacker/commands/stacker/diff.py

```diff
@@ -0,0 +1,35 @@
+""" Diffs the config against the currently running CloudFormation stacks
+
+Sometimes small changes can have big impacts.  Run 'stacker diff' before
+'stacker build' to detect bad things(tm) from happening in advance!
+"""
+
+from .base import BaseCommand
+from ...actions import diff
+
+
+class Diff(BaseCommand):
+    name = 'diff'
+    description = __doc__
+
+    def add_arguments(self, parser):
+        super(Diff, self).add_arguments(parser)
+        parser.add_argument("--force", action="append", default=[],
```

Comment:
Yeah, just got back to that - it's kind of a weird syntax here. This really makes me wish we had 'reasons' attached to SKIPPED results.